simplesource-data/src/test/java/io/simplesource/data/FutureResultTest.java

```diff
@@ -1,167 +0,0 @@
-package io.simplesource.data;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-import java.util.List;
-import java.util.concurrent.CountDownLatch;
-import java.util.function.Function;
-import java.util.stream.Collectors;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-class FutureResultTest {
```

Comment:
I remove these test just because I didn't want to bother to make them compile again.